xbmc/interfaces/python/AddonPythonInvoker.cpp

```diff
@@ -84,45 +85,31 @@ PyObject* PyInit_Module_xbmcvfs(void);
 
 using namespace PythonBindings;
 
-typedef struct
+namespace
 {
-  const char *name;
-  CPythonInvoker::PythonModuleInitialization initialization;
-} PythonModule;
-
-static PythonModule PythonModules[] =
+// clang-format off
+const std::array<PythonModule, 6> PythonModules =
   {
-    { "xbmcdrm",    PyInit_Module_xbmcdrm    },
-    { "xbmcgui",    PyInit_Module_xbmcgui    },
-    { "xbmc",       PyInit_Module_xbmc       },
-    { "xbmcplugin", PyInit_Module_xbmcplugin },
-    { "xbmcaddon",  PyInit_Module_xbmcaddon  },
-    { "xbmcvfs",    PyInit_Module_xbmcvfs    }
+    PythonModule{ "xbmcdrm",    PyInit_Module_xbmcdrm    },
```

Comment:
please don't use whitespace formatting, you can leave it to clang-format unless there really is a readability issue.

Reply:
Removed.